pandas/tools/util.py

```diff
@@ -238,3 +238,39 @@ def to_numeric(arg, errors='raise', downcast=None):
         return values[0]
     else:
         return values
+
+
```

Comment:
this is not the right place for this at all. remove all of this added code from tools/util (and the tests)

```
In [2]: pd.DataFrame()._get_axis_number(1)
Out[2]: 1

In [3]: pd.DataFrame()._get_axis_number('index')
Out[3]: 0

In [4]: pd.DataFrame()._get_axis_number('columns')
Out[4]: 1

In [5]: pd.DataFrame()._get_axis_number(1)
Out[5]: 1
```